### ISSUE TITLE
Add special rules for Scenario 118

### ISSUE THREAD

**Issue by Jaerin** [Lurkars/gloomhavensecretariat#550]
### Is your feature request related to a problem? Please describe.

Scenario 118 has a couple of special rules for the monsters that get spawned.

### Describe the solution you'd like

Odd number standees for Living Bones and Living Corpses should have +3 shield applied to them
The even number standees for living bone/living corpses need a way to keep track of number tokens
If possible when the living bones turn is executed it should check if any even numbered living bones or frozen corpsed have been killed and request if a normal living bones should be spawned with that numbered token.  

### Describe alternatives you've considered

s

**Comment by Jaerin** [comment]
Understand its not a straightforward scenario by any means.
The +3 shields to odd standees by default would probably be a nice QOL start for it even if it couldn't be implemented fully.

**Comment by Lurkars** [comment]
yes, as start now I implemented that automatic assignment to the shield or normal monsters based on standee number (hope I understood the rules right!). So happy about first feedback on [v0.95.10](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.95.10) here.

**Comment by Jaerin** [comment]
I agree the rules are not very intuitively clear about how its supposed to work.  Like the idea that the respawned mob is a living bones which means you could end up with an odd standee with an even number.

**Comment by rdoll** [comment]
Just played this scenario and the support mostly worked. We had several occurrences of a mob that would be grayed out instead of getting its turn, which annoyingly caused any effects it had to not advance. Here's an example of a round where the Living Bones went first, but it started gray after the round starting Draw (GHS version 0.99.21 latest on Chrome).

![image](https://github.com/user-attachments/assets/e829b509-8ec5-4e1f-8655-ff70532f69f8)

We also had a few cases of where mobs were not grayed out, but they didn't draw an action card. Sorry, no image for that one, but I'm guessing this is related to the first issue.

Above aside, the approach worked well with one exception: after looting the protection amulet from an enemy, there was no way to decrease the shield of the "#118" mob. FWIW, we put a summon token on it to indicate "no longer has an amulet".

**Comment by Lurkars** [comment]
Thanks for noticing the issue with the disabled monsters. Hopefully fixed in  [v0.99.22](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.99.22). Keep the issue open, because yes, the special rules are still not implemented fully/correctly.
